### PR TITLE
[Merged by Bors] - Add AuthenticationClass::resolve

### PR DESCRIPTION
**Pull request by sbernauer**
## Description

*Please add a description here. This will become the commit message of the merge request later.*

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Add `AuthenticationClass::resolve` helper function ([#432]).
+
+[#432]: https://github.com/stackabletech/operator-rs/pull/408
```
Comment (maltesander) [inline CHANGELOG.md:11]:
```suggestion
[#432]: https://github.com/stackabletech/operator-rs/pull/432
```

**Review by maltesander — CHANGES_REQUESTED** [review]
One minor thing. LGTM otherwise.

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by sbernauer** [comment]
thx!

**Pull request closed without merging**